### ISSUE TITLE
API is shutting down

### ISSUE THREAD

**Issue by erlendsellie** [tormjens/home-assistant-sbanken#4]
Thanks for this integration. It was fun while it lasted.

![Screenshot_20240124-223406](https://github.com/tormjens/home-assistant-sbanken/assets/18458417/6a8fecee-3b22-4538-b9ee-72387a3e00ba)

Hopefully the app can be reversed engineered or something.

**Comment by tormjens** [comment]
Yeah, I saw. Seems that "nothing will change for Sbanken customers' was a bit of a lie. :( 

Bye, bye.
